src/OrleansRuntime/Silo/Silo.cs

```diff
@@ -166,7 +166,6 @@ internal Silo(string name, SiloType siloType, ClusterConfiguration config, ILoca
             StatisticsCollector.Initialize(nodeConfig);
             
             SerializationManager.Initialize(globalConfig.UseStandardSerializer, globalConfig.SerializationProviders, globalConfig.UseJsonFallbackSerializer);
-            CodeGeneratorManager.GenerateAndCacheCodeForAllAssemblies();
```

Comment:
The line is redundant (not present  in client initialization either) - the call is already made as part of the `SerializationManager` [initialization](https://github.com/ca-ta/orleans/blob/codegen_initialization/src/OrleansRuntime/Silo/Silo.cs#L168), [here](https://github.com/ca-ta/orleans/blob/codegen_initialization/src/Orleans/AssemblyLoader/AssemblyProcessor.cs#L70).